<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>path_load_save_selector</class>
 <widget class="QWidget" name="path_load_save_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="file_line_edit" name="txtLoad">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string> File</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSave">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Save</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="slim5.qrc">
       <normaloff>:/images/disk.svg</normaloff>:/images/disk.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnLoad">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Load</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="slim5.qrc">
       <normaloff>:/images/folder-open.svg</normaloff>:/images/folder-open.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+L</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>file_line_edit</class>
   <extends>QLineEdit</extends>
   <header>file_line_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="slim5.qrc"/>
 </resources>
 <connections/>
</ui>
